README.md

```diff
@@ -65,7 +65,7 @@ cmake \
   -DIREE_CMAKE_PLUGIN_PATHS=$PWD \
   -DIREE_BUILD_PYTHON_BINDINGS=ON \
   -DIREE_INPUT_STABLEHLO=OFF \
-  -DIREE_INPUT_TORCH=OFF 
+  -DIREE_INPUT_TORCH=OFF \
```

Comment:
I know this is a big nit, but you can just create a separate PR for these changes as well.

Reply:
Moved to #934